exercises/variable-length-quantity/src/test/java/VariableLengthQuantityTest.java

```diff
@@ -0,0 +1,262 @@
+       List<String> expected = Arrays.asList("0xff", "0xff", "0x7f");
+       List<Long> numbers = Arrays.asList(0x1fffffL);
+
+       assertEquals(expected, variableLengthQuantity.encode(numbers));
+    }
+
+    @Ignore("Remove to run test")
+    @Test
+    public void testSmallestQuadrupleByte() {
+        List<String> expected = Arrays.asList("0x81", "0x80", "0x80", "0x0");
+        List<Long> numbers = Arrays.asList(0x200000L);
+
+        assertEquals(expected, variableLengthQuantity.encode(numbers));
+    }
+
+    @Ignore("Remove to run test")
+    @Test
+    public void testArbitraryQuadrupleByte() {
+        List<String> expected = Arrays.asList("0xc0", "0x80", "0x80", "0x0");
+        List<Long> numbers = Arrays.asList(0x8000000L);
+
+        assertEquals(expected, variableLengthQuantity.encode(numbers));
+    }
+
+    @Ignore("Remove to run test")
+    @Test
+    public void testLargestQuadrupleByte() {
+        List<String> expected = Arrays.asList("0xff", "0xff", "0xff", "0x7f");
+        List<Long> numbers = Arrays.asList(0xfffffffL);
+
+        assertEquals(expected, variableLengthQuantity.encode(numbers));
+    }
+
+    @Ignore("Remove to run test")
+    @Test
+    public void testSmallestQuintupleByte() {
+        List<String> expected = Arrays.asList("0x81", "0x80", "0x80", "0x80", "0x0");
+        List<Long> numbers = Arrays.asList(0x10000000L);
+
+        assertEquals(expected, variableLengthQuantity.encode(numbers));
+    }
+
+    @Ignore("Remove to run test")
+    @Test
+    public void testArbitraryQuintupleByte() {
+        List<String> expected = Arrays.asList("0x8f", "0xf8", "0x80", "0x80", "0x0");
+        List<Long> numbers = Arrays.asList(0xff000000L);
+
+        assertEquals(expected, variableLengthQuantity.encode(numbers));
+    }
+
+    @Ignore("Remove to run test")
+    @Test
+    public void testMaximum32BitIntegerInput() {
+        List<String> expected = Arrays.asList("0x8f", "0xff", "0xff", "0xff", "0x7f");
+        List<Long> numbers = Arrays.asList(0xffffffffL);
+
+        assertEquals(expected, variableLengthQuantity.encode(numbers));
+    }
+
+    @Ignore("Remove to run test")
+    @Test
+    public void testTwoSingleByteValues() {
+        List<String> expected = Arrays.asList("0x40", "0x7f");
+        List<Long> numbers = Arrays.asList(0x40L, 0x7fL);
+
+        assertEquals(expected, variableLengthQuantity.encode(numbers));
+    }
+
+    @Ignore("Remove to run test")
+    @Test
+    public void testTwoMultiByteValues() {
+        List<String> expected = Arrays.asList("0x81", "0x80", "0x0", "0xc8", "0xe8", "0x56");
+        List<Long> numbers = Arrays.asList(0x4000L, 0x123456L);
+
+        assertEquals(expected, variableLengthQuantity.encode(numbers));
+    }
+
+    @Ignore("Remove to run test")
+    @Test
+    public void testManyMultiByteValues() {
+        List<String> expected = Arrays.asList("0xc0", "0x0", "0xc8", "0xe8",
+        "0x56", "0xff", "0xff", "0xff", "0x7f", "0x0", "0xff", "0x7f", "0x81",
```

Comment:
Very minor nitpick: this should be indented so that it's obvious that it's part of the same command as the previous line, if that makes sense 🙂 Common best practice is to put each argument on a new line if they don't all fit on one line. But that would make for a lot of lines in this case, so it's up to you whether you want to stick to that or not 🙂 

The same goes for the other places with line breaks 🙂